apps/docs/spec/supabase_kt_v2.yml

```
@@ -4393,7 +4393,10 @@ functions:
         isSpotlight: true
         code: |
           ```kotlin
-          supabase.functions.invoke("function_name")
+          val response = supabase.functions.invoke("function_name")
+
+          // Decode the response body to a serializable class
+          val data = res.body<FunctionResponse>()
```

```suggestion
          val response = supabase.functions.invoke("function_name")

          // Decode the response body to a serializable class
          val data = response.body<FunctionResponse>()
```
If we expand the snippet here, should we also do this with the other function snippets? Also, if we introduce the `body` function, we should probably add a comment in the description. Maybe something like "The return type of the invoking methods is an `HttpResponse`. You can access the response body via `response.body<T>()`, where T can be any serializable type."

---

>If we expand the snippet here, should we also do this with the other function snippets?

When we do add more functions examples, we probably should add these decoding examples wherever appropriate. For now, I think we are good.